src/components/UI/ResourceEvent/ResourceEvent.tsx

```diff
@@ -1,117 +1,62 @@
-import { LinkButton } from "@govtechsg/tradetrust-ui-components";
-import { isFuture, format } from "date-fns";
+import { format } from "date-fns";
 import React, { FunctionComponent } from "react";
-import { ExternalLink, PlayCircle } from "react-feather";
 import { formatTime } from "../../../common/utils/dateTime";
+import { Link } from "react-router-dom";
 
-export interface EventProps {
-  attributes: {
-    title: string;
-    thumbnail?: string;
-    blurb: string;
-    link: string;
-    date: string;
-    timeStart: string;
-    timeEnd: string;
-    videoLink?: string;
-    slides?: string;
-    registerLink?: string;
-  };
+export type EventProps = {
+  slug: string;
+  attributes: EventAttribute;
+};
+export interface EventAttribute {
```

Comment:
dont export if its a local interface

Reply:
ok my bad forgot to remove it, was using it some place else before.